components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Extensions/PublishedNodesEntryModelEx.cs

```diff
@@ -58,6 +58,69 @@ public static class PublishedNodesEntryModelEx {
             return true;
         }
 
+        /// <summary>
+        /// filter model
+        /// </summary>
+        public static bool IsFiltered(this PublishedNodesEntryModel model, PublishedNodesEntryModel that) {
```

Comment:
>IsFiltered [](http://example.com/codeflow?start=27&length=10)

what is the purpose of this method? how is it different from HasSameGroup()?

Reply:
the differences are:
added comparison also for DataSetWriterId
added for each element also the check that is not null or default (OpcAuthenticationMode)